src/Catty/DataModel/Bricks/Look/PreviousLookBrick.swift

```diff
@@ -20,27 +20,40 @@
  *  along with this program.  If not, see http://www.gnu.org/licenses/.
  */
 
-#import "PreviousLookBrickCell.h"
+@objc(PreviousLookBrick)
+@objcMembers class PreviousLookBrick: Brick, BrickProtocol {
+    @objc var look: Look?
 
-@interface PreviousLookBrickCell ()
-@property (nonatomic, strong) UILabel *textLabel;
-@end
+    init(look: Look) {
+        self.look = look
+        super.init()
+    }
 
-@implementation PreviousLookBrickCell
+    public required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
 
-+ (CGFloat)cellHeight
-{
-    return kBrickHeight1h;
-}
+    override public required init() {
+        super.init()
+    }
 
-- (void)hookUpSubViews:(NSArray *)inlineViewSubViews
-{
-    self.textLabel = inlineViewSubViews[0];
-}
+    func path(for look: Look?) -> String? {
```

Comment:
Please make the `look` parameter non-optional, since that method would not make sense without a look being passed.